docs/src/content/docs/concepts/context.mdx

```diff
@@ -188,9 +188,53 @@ In this example, the `WebPageSummarizer` component can access both the OpenAI cl
 
 The order of nesting doesn't matter as long as the component using a context is wrapped by its corresponding provider somewhere up the tree.
 
-## Additional Resources
+### Using multiple OpenAI compatible providers
+
+You can also nest multiple providers to use LLMs from different labs in the same workflow. When you nest multiple OpenAIproviders, components will always use the closest provider in the tree. The example below shows how you could use models from OpenAI and Groq in the same workflow using their OpenAI-compatible APIs.
```

Comment:
Nit: not sure if it's worth clarifying closest provider vs closest provider above it in the tree.

Reply:
updated